applauncher/src/main/java/de/interseroh/tmb/applauncher/client/ApplauncherWebAppGinjector.java

```diff
@@ -23,7 +23,7 @@
 
 import de.interseroh.tmb.applauncher.client.common.ServicePreparator;
 
-@GinModules(ApplanucherWebAppGinModul.class)
-public interface ApplanucherWebAppGinjector extends Ginjector {
+@GinModules(ApplauncherWebAppGinModul.class)
+public interface ApplauncherWebAppGinjector extends Ginjector {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Annotate the "ApplauncherWebAppGinjector" interface with the @FunctionalInterface annotation [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1609)